src/lavinmq/federation/link.cr

```diff
@@ -28,6 +29,8 @@ module LavinMQ
           uri = @upstream.uri
           ui = uri.userinfo
           @scrubbed_uri = ui.nil? ? uri.to_s : uri.to_s.sub("#{ui}@", "")
+          @connection_information = ::AMQP::Client::ConnectionInformation.new(
```

Comment:
Is `platform` really relevant here? 

This can be moved to line 20.

Reply:
No, only product+product version